resteasy-core/src/main/java/org/jboss/resteasy/core/providerfactory/DefaultExceptionMapper.java

```diff
@@ -20,19 +20,24 @@
 package org.jboss.resteasy.core.providerfactory;
 
 import jakarta.ws.rs.WebApplicationException;
+import jakarta.ws.rs.container.ResourceInfo;
+import jakarta.ws.rs.core.Request;
 import jakarta.ws.rs.core.Response;
+import jakarta.ws.rs.core.UriInfo;
 import jakarta.ws.rs.ext.ExceptionMapper;
 
 import org.jboss.logging.Logger;
+import org.jboss.resteasy.resteasy_jaxrs.i18n.LogMessages;
 import org.jboss.resteasy.spi.ApplicationException;
+import org.jboss.resteasy.spi.ResteasyProviderFactory;
 
 /**
  * The default {@link ExceptionMapper} for RESTEasy.
  *
  * @author <a href="mailto:jperkins@redhat.com">James R. Perkins</a>
  */
```

Comment:
Shouldn't this have a minimal priority?

Reply:
It doesn't have a priority because it is never looked up.